src/Esfa.Vacancy.Register.Infrastructure/Repositories/VacancyRepository.cs

```diff
@@ -47,27 +47,58 @@ public VacancyRepository(IProvideSettings provideSettings)
 ,       V.WorkingWeek
 ,       V.WageText
 ,       V.HoursPerWeek
-,       V.DurationValue AS ExpectedDuration
+,       V.ExpectedDuration 
 ,       V.ExpectedStartDate
 ,		VH.HistoryDate AS DatePosted
 ,       V.ApplicationClosingDate
 ,       V.NumberofPositions 
 ,       V.EmployerDescription
 ,       V.EmployersWebsite
 ,       V.TrainingTypeId
-,       RS.LarsCode AS LarsStandardId
-,       CAST(AO.ShortName AS INT) AS SSA1Code
+,       RS.LarsCode AS StandardCode
+,       AF.ShortName AS FrameworkCode
+,       E.FullName AS EmployerName
+,       V.EmployerDescription
+,       V.EmployersWebsite
+,       TextFields.[TrainingToBeProvided]
+,       TextFields.[QulificationsRequired]
+,       TextFields.[SkillsRequired]
+,       TextFields.[PersonalQualities]
+,       TextFields.[ImportantInformation]
+,       TextFields.[FutureProspects]
+,       TextFields.[RealityCheck]
 FROM[dbo].[Vacancy]        V
-INNER JOIN [dbo].[VacancyHistory] VH
-	ON V.VacancyId = VH.VacancyId and VH.VacancyHistoryEventSubTypeId = 2
+INNER JOIN (SELECT VacancyId, Min(HistoryDate) HistoryDate
+            FROM [dbo].[VacancyHistory]
+            WHERE VacancyHistoryEventTypeId = 1
+            AND VacancyHistoryEventSubTypeId = 2
+            GROUP BY VacancyId
+           ) VH
+	ON V.VacancyId = VH.VacancyId 
 LEFT JOIN   [Reference].[Standard] RS 
     ON V.StandardId = RS.StandardId
-LEFT JOIN   [Reference].[StandardSector] RSS
-    ON RS.StandardSectorId = RSS.StandardSectorId
-LEFT JOIN    [dbo].[ApprenticeshipOccupation] AO
-    ON RSS.ApprenticeshipOccupationId = AO.ApprenticeshipOccupationId
+LEFT JOIN   [dbo].[ApprenticeshipFramework] AF
+    ON      V.ApprenticeshipFrameworkId = AF.ApprenticeshipFrameworkId
+INNER JOIN VacancyOwnerRelationship AS R 
```

Comment:
No problem with using aliases declared with the 'AS' statement but we should be consistent with it elsewhere in the query or remove it.